src/omotes_sdk/internal/worker/worker.py

```diff
@@ -228,8 +228,6 @@ def wrapped_worker_task(
 class Worker:
     """Worker which works as an Celery Worker application and runs a single type of task."""
 
-    config = WorkerConfig()
```

Comment:
Suggestion: Add `config: WorkerConfig` to keep the type annotation so it conforms to pep 526

Reply:
Added the config type in the new commit.